### PR TITLE
Only show travel costs grant if UK full time student

### PR DESCRIPTION
**Pull request by thomasleese**
You can't get the grant if you're not a UK full time student.

[Trello Card](https://trello.com/c/nt7oYzwa/711-student-finance-calculator-remove-travel-costs-from-part-time-outcomes)

### CODE REVIEW

**Comment by thomasleese** [comment]
@edwardkerry This should all pass now!

File: lib/smart_answer_flows/student-finance-calculator/outcomes/outcome_uk_all_students.govspeak.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
     To qualify for a Maintenance Loan you must be studying an Initial Teacher Training course (if it's degree level or above).
   <% elsif course_studied == 'none-of-the-above' %>
     To qualify for a Maintenance Loan, you must be studying one of the following courses:
-    
+
```
Comment (edwardkerry) [inline lib/smart_answer_flows/student-finance-calculator/outcomes/outcome_uk_all_students.govspeak.erb]: is this line intentional? :)

Comment (thomasleese) [reply lib/smart_answer_flows/student-finance-calculator/outcomes/outcome_uk_all_students.govspeak.erb]: My editor deleting the trailing whitespace for me! I tend to leave them in.

**Comment by bevanloon** [comment]
screen shot 2019-01-17 at 08 36 43
🎉

**Pull request merged**